config/v0.3.0/propagation.go

```diff
@@ -0,0 +1,60 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package config // import "go.opentelemetry.io/contrib/config/v0.3.0"
+
+import (
+	"errors"
+
+	"go.opentelemetry.io/contrib/propagators/aws/xray"
+	"go.opentelemetry.io/contrib/propagators/b3"
+	"go.opentelemetry.io/contrib/propagators/jaeger"
+	"go.opentelemetry.io/contrib/propagators/ot"
+	"go.opentelemetry.io/otel/propagation"
+)
+
+func propagator(cfg configOptions) (propagation.TextMapPropagator, error) {
+	if cfg.opentelemetryConfig.Propagator == nil {
+		return propagation.NewCompositeTextMapPropagator(), nil
```

Comment:
Should we not default to `tracecontext,baggage`?
PTAL @open-telemetry/configuration-approvers

Reply:
I'm not so sure: that's not the default behavior of an unconfigured SDK, from what I remember. Having a default set of propagators would deviate from that behavior.

Reply:
This should default to the same as the env variables defined in the spec: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/sdk-environment-variables.md#general-sdk-configuration